libcodechecker/libhandlers/server.py

```diff
@@ -247,6 +256,47 @@ def add_arguments_to_parser(parser):
                                 help="Stops all of your running CodeChecker "
                                      "server instances.")
 
+    database_mgmnt = parser.add_argument_group(
+            "Database management arguments.",
+            """WARNING these commands needs to be called with the same
+            workspace and configuration arguments as the server so the
+            configuration database will be found which is required for the
+            schema migration. Migration can be done without a running server
+            but pay attention to use the same arguments which will be used to
+            start the server.
+            NOTE:
+            Before migration it is advised to create a full a backup of
+            the product databases.
+            """)
+
+    database_mgmnt = database_mgmnt. \
+        add_mutually_exclusive_group(required=False)
+
+    database_mgmnt.add_argument('--status',
```

Comment:
Update the user guide with these new arguments: https://github.com/Ericsson/codechecker/blob/master/docs/user_guide.md#server